src/main/java/christmas/validation/DomainValidator.java

```diff
@@ -0,0 +1,28 @@
+package christmas.validation;
+
+import christmas.domain.constant.Menu;
+import java.util.Map;
+
+public class DomainValidator {
+
+	private static final int EVENT_START = 1;
+	private static final int EVENT_END = 31;
+
+	public static boolean validateBookingDate(int date) {
+		return (date < EVENT_START || date > EVENT_END);
+	}
+
+	public static boolean validateBeverageOnly(Map<String, Integer> userOrder) {
+		int countNonBeverage = (int) userOrder.keySet().stream()
+				.filter(Menu::isNotBeverage)
+				.count();
+		return countNonBeverage == 0;
+	}
```

Comment:
일단 int로 한번 변환해줄 필요가 없는 것 같아요.

그리고 noneMatch를 사용하면 좀 더 간결히 쓸 수 있을 것 같아요
```suggestion
    public static boolean validateBeverageOnly(Map<String, Integer> userOrder) {
        return userOrder.keySet().stream()
                .noneMatch(Menu::isNotBeverage);
    }
```